proposals/utils/stepper_helpers.py

```diff
@@ -251,24 +282,38 @@ def instantiate_form(self):
         )
         return model_form
 
-    def get_errors(self):
+    def get_errors(self, include_children=False):
         """
-        By default, this returns the errors of the instantiated form, which get
-        evaluated as a boolean. By passing a custom get_checker_errors function
-        from the checker, custom errors can be specified also.
+        By default, this returns the errors of the instantiated form and,
+        optionally, its children. This then gets evaluated as a boolean.
+        By passing a custom get_checker_errors function
+        from the checker, custom errors also be added.
         """
-        if hasattr(self, "get_checker_errors"):
-            return self.get_checker_errors() or self.model_form.errors
-        return self.model_form.errors
+        errors = []
+        errors += self.model_form.errors
 
-    def css_classes(self):
-        css_classes = super().css_classes()
+        if hasattr(self, "get_checker_errors"):
+            errors += self.get_checker_errors()
+        if include_children and self.children:
+            for child in self.children:
+                errors += child.get_errors(include_children=True)
+        return errors
```

Comment:
No, I don't think so because the super() just goes to StepperItem's get_errors which just returns an empty list. This is where the behaviour of this function gets specified. Up the chain, include children becomes basically meaningless.